Documentation/contributing/development/debugging.rst

```diff
@@ -4,10 +4,69 @@
     Please use the official rendered version released here:
     https://docs.cilium.io
 
+.. _gs_debugging:
+
 #########
 Debugging
 #########
 
+Attaching a debugger
+--------------------
+
+Cilium comes with a handy set of Makefile targets for quickly deploying
```

Comment:
```suggestion
Cilium comes with a useful set of Makefile targets for quickly deploying
```

Can we replace `handy` with `useful` to help users who may not be familiar with this term?

Reply:
Strange, but ok